base/base_tests/string_utils_test.cpp

```diff
@@ -165,6 +165,10 @@ UNIT_TEST(to_double)
 
   s = "123.456 we don't parse it.";
   TEST(!strings::to_double(s, d), ());
+
+  TEST(!strings::to_double("INF", d), ());
+  TEST(!strings::to_double("NAN", d), ());
+  TEST(!strings::to_double((ostringstream() << numeric_limits<long double>::max()).str(), d), ());
```

Comment:
The size of long double is not specified, as far as I know. I.e. it can be that long double is the same as double.